ui/admin/tests/acceptance/onboarding/success-test.js

```diff
@@ -0,0 +1,30 @@
+import { module, test } from 'qunit';
+import { visit, currentURL, click } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
+import { authenticateSession } from 'ember-simple-auth/test-support';
+
+module(
+  'Acceptance | onboarding | quick-setup | create-resources | success',
+  function (hooks) {
+    setupApplicationTest(hooks);
+    setupMirage(hooks);
+
+    hooks.beforeEach(function () {
+      authenticateSession({});
+    });
+
+    const urls = {
+      createResources:
+        '/scopes/global/onboarding/quick-setup/create-resources/success',
+      targets: '/scopes/global/targets',
+    };
+
+    test.skip('clicking done takes user to targets screen', async function (assert) {
```

Comment:
skipping this test for now, but this will be included in a separate when the button is included in the template